packages/gitmoji-changelog-core/src/index.js

```diff
@@ -57,8 +58,17 @@ function sanitizeVersion(version) {
   })
 }
 
+function groupCommitsByMessagesDistance(commits = []) {
```

Comment:
I find the method name too technical. What do you think of groupSimilarCommits ?

Reply:
Ok